pandas/tests/scalar/timedelta/test_construction.py

```diff
@@ -89,6 +89,9 @@ def test_construction():
     with pytest.raises(ValueError):
         Timedelta('3.1415')
 
+    with pytest.raises(ValueError):
```

Comment:
add a comment why this is raising (add to the other cases if you can)

Reply:
can you test the error message that is returned here (assert_raises_regex)

Reply:
actually this case is kind of duplicative of your new test below)

Reply:
I will remove everything here and add the `3.1415` case to the new test case.